taskcluster/xpi_taskgraph/release_promotion.py

```diff
@@ -101,6 +101,14 @@ def is_release_promotion_available(parameters):
                 "description": ("The expected version to promote."),
                 "default": "",
             },
+            "additional_shipit_emails": {
+                "type": "array",
+                "description": (
+                    "Optional: an array of email strings that "
+                    " should be notified when releases change status"
```

Comment:
Nit: we may get two spaces in a row here between "that" and "should", no? Not sure if we normalize spaces here or not.

Reply:
You might be right - fixed.